assets/js/components/Dashboard/PlaceRowAccordion.tsx

```diff
@@ -15,20 +15,22 @@ import ScreenDetail from "Components/ScreenDetail";
 import { sortScreens } from "../../util";
 import { useUpdateAnimation } from "Hooks/useUpdateAnimation";
 import classNames from "classnames";
+import _ from "lodash";
```

Comment:
I thought we were using `lodash/fp`?

Reply:
Ahh yes thank you. (Although I dislike how bad it is at inferring types 😅)

Reply:
I agree this is a notable downside of FP and maybe something we should have considered (if we'd had a better sense for it at the time) in our conversations about which "utility library" to use. I tend to avoid Lodash entirely and use vanilla JS where feasible, since built-in classes and functions have generally excellent types (except for you, `JSON.parse`, you're not invited to the party).